opentelemetry-sdk/src/resource/mod.rs

```diff
@@ -169,11 +169,11 @@ impl Resource {
     /// 5. If both resources do not have a schema url, the schema url will be empty.
     ///
     /// [Schema url]: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.9.0/specification/schemas/overview.md#schema-url
-    fn merge<T: Deref<Target = Self>>(&self, other: T) -> Self {
-        if self.is_empty() {
+    pub(crate) fn merge<T: Deref<Target = Self>>(&self, other: T) -> Self {
```

Comment:
This is also a breaking change, and good to explicitly mention in the changelog.

Reply:
And this should be made crate-private only once `with_resource` is made additive for all the signals.

Reply:
The `merge` function was private before and now I just made it internal to the crate. This should not cause any breaking changes in public API, or am I missing something?

Reply:
Yes, sorry for confusion.